# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1237293789 -3600
#      Tue Mar 17 13:43:09 2009 +0100
# Node ID c235a59d30a4a072889f973893914de3f1866aff
# Parent  349d7516167203248b9fae25eec9e46b19ff91e7
remove dead code from fsolve

diff --git a/scripts/optimization/fsolve.m b/scripts/optimization/fsolve.m
--- a/scripts/optimization/fsolve.m
+++ b/scripts/optimization/fsolve.m
@@ -252,24 +252,16 @@ function [x, fvec, info, output, fjac] =
     ## of course, how to define the above terms :)
 
     lastratio = 0;
     nfail = 0;
     nsuc = 0;
     ## Inner loop.
     while (niter <= maxiter && nfev < maxfev && ! info)
 
-      if (useqr)
-        tr_mat = r;
-        tr_vec = q'*fvec;
-      else
-        tr_mat = fjac;
-        tr_vec = fvec;
-      endif
-
       ## Get trust-region model (dogleg) minimizer.
       if (useqr)
         qtf = q'*fvec;
         s = - __dogleg__ (r, qtf, dg, delta);
         w = qtf + r * s;
       else
         s = - __dogleg__ (fjac, fvec, dg, delta);
         w = fvec + fjac * s;
